docs/api-reference/overview.mdx

```diff
@@ -5,6 +5,17 @@ description: "The front-end client (e.g. browser) creates, reads, updates, and d
 resources by using the backend server’s API."
 ---
 
+## Quickstart
+
+Want to jump-in and interact with Mage via API? 
+
+Check out our quick start for a Jupyter notebook to authenticate with Python Requests and hit the ground running _or_ read more about our API structure— the compenents that underpin functionality.
+
+<CardGroup>
+<Card title="Quick start" icon="person-running" href="https://github.com/mage-ai/mage-ai/blob/master/templates/api-reference/quickstart.ipynb"></Card>
```

Comment:
I think we can just provide python code. No need to provide the jupyter notebook here.
They can also just run these code in Mage.